includes/multi-currency/MultiCurrency.php

```diff
@@ -845,6 +845,45 @@ public function get_price( $price, string $type ): float {
 		return $this->get_adjusted_price( $converted_price, $apply_charm_pricing, $currency );
 	}
 
+	/**
+	 * Gets a raw converted amount based on the amount and currency codes passed.
+	 * This is a helper method for external conversions, if needed.
+	 *
+	 * @param float  $amount        The amount to be converted.
+	 * @param string $to_currency   The 3 letter currency code to convert the amount to.
+	 * @param string $from_currency The 3 letter currency code to convert the amount from.
+	 *
+	 * @return float The converted amount.
+	 */
+	public function get_raw_conversion( float $amount, string $to_currency, string $from_currency = '' ): float {
+		$enabled_currencies = $this->get_enabled_currencies();
+
+		// We return the amount if the to_currency is not enabled.
+		$to_currency = strtoupper( $to_currency );
+		if ( ! isset( $enabled_currencies[ $to_currency ] ) ) {
+			return $amount;
+		}
+
+		if ( '' !== $from_currency ) {
```

Comment:
This could be simplified with setting `$from_currency_rate` to `1` when the `$from_currency` is empty. And then the converted `$amount` would be `$amount * ($to_currency_rate / $from_currency_rate). 

Also we could add a `0` check to the `$from_currency_rate` to prevent fatal errors.